<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>串口测试</string>
       </attribute>
       <widget class="QTextEdit" name="serial_recv_data_text_">
        <property name="geometry">
         <rect>
          <x>293</x>
          <y>10</y>
          <width>471</width>
          <height>241</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>137</width>
          <height>185</height>
         </rect>
        </property>
        <property name="title">
         <string>串口设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>端口</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="serial_port_cmb_">
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="baudrates_cmb_">
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1" rowspan="2">
          <widget class="QComboBox" name="data_bits_cmb_">
           <property name="currentIndex">
            <number>3</number>
           </property>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>数据位</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>校验位</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="parity_cmb_">
           <item>
            <property name="text">
             <string>无校验</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>奇校验</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>偶校验</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>停止位</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="stop_bit_cmb_">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="serial_btn_">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QWidget" name="serial_status_" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="serial_clear_btn_">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>120</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="serial_send_text_">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>290</y>
          <width>471</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="serial_send_btn_">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>390</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>191</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>遥控解码</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>CH1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ch1_num_">
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
             <property name="value" stdset="0">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>CH2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ch2_num_">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>CH3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ch3_num_">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>CH4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ch4_num_">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>CH5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ch5_num_">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>CH6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ch6_num_">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>CH7</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="ch7_num_">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>CH8</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLCDNumber" name="ch8_num_">
             <property name="font">
              <font>
               <family>3ds</family>
              </font>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>CAN测试</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>211</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>CAN口设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>设备类型</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox_3"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>USB驱动</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="can_open_btn_">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>USBCAN-2A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>USBCAN-2C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CANalyst-II</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QWidget" name="serial_status_1" native="true"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="can_cmb_"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>CAN 口</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_4"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>网络测试</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>341</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string>网络设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="1" column="2">
          <widget class="QLineEdit" name="nettest_local_port_text_">
           <property name="text">
            <string>20201</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="nettest_mode_cbox_">
           <property name="currentText">
            <string>UDP 客户端</string>
           </property>
           <item>
            <property name="text">
             <string>UDP 客户端</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UDP 服务器</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TCP 客户端</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TCP 服务器</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="nettest_remote_port_text_">
           <property name="text">
            <string>20202</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="nettest_open_btn_">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="nettest_remote_ip_text_">
           <property name="text">
            <string>192.168.1.39</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="nettest_local_port_label_">
           <property name="text">
            <string>本地端口</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="nettest_remote_ip_label_">
           <property name="text">
            <string>远端IP</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QWidget" name="nettest_status_" native="true"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>模式</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="nettest_remote_port_label_">
           <property name="text">
            <string>远端端口</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextEdit" name="nettest_text_">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>250</y>
          <width>751</width>
          <height>201</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTextEdit" name="nettest_send_dat_text_">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>30</y>
          <width>381</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="nettest_send_btn_">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>210</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QPushButton" name="nettest_clear_btn_">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>470</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
       <widget class="QPushButton" name="nettest_connect_btn_">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>210</y>
          <width>267</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>  连接</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
